archive/f/ferret/hello-world.clj

```diff
@@ -0,0 +1,3 @@
+(do
+  (println "Hello")
```

Comment:
Any particular reason for the println? For consistency, we stick with "Hello, World!".

Reply:
I saw Hello world in Java and it used a `println`. Hence, I used that too.

I'll update the text to Hello, World.

Reply:
Sorry, I meant the string more than the usage. Println is fine!